openlibrary/plugins/recaptcha/recaptcha.py

```diff
@@ -36,24 +17,17 @@ def __init__(self, public_key, private_key):
 
         self.error = None
 
-    def render(self):
```

Comment:
Thanks for the great start @salman-bhai ! This `render()` is used by edit pages though to conditionally add Recaptcha to  pages based on user type. e.g.  openlibrary/plugins/upstream/addbook.py and openlibrary/templates/books/add.html + openlibrary/templates/books/edit.html
https://github.com/internetarchive/openlibrary/blob/f8092840a77c7479a352fd83fd068d340f97d3e3/openlibrary/templates/books/add.html#L74

A search for all the places referencing `recaptcha`: https://github.com/internetarchive/openlibrary/search?p=1&q=recaptcha&type=&utf8=%E2%9C%93

Admin users should never see a recaptcha on an edit page, and users who have been members for more than two years should not either (see https://github.com/internetarchive/openlibrary/blob/85dd7cd522d34f7e06e6c777e437b64becfcc0da/openlibrary/plugins/upstream/addbook.py#L43)  Testing all the cases in dev is a little tricky :)

Reply:
Also I propose we delete this file if it's not being used on the Production server as well. I don't see the use of it atleast for recaptcha!

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/recaptcha/code.py